config/sidebars.js

```diff
@@ -320,7 +322,7 @@ const sidebars = {
     {
       type: "doc",
       id: "cli/README",
-      label: "CLI",
+      label: "Command Line",
```

Comment:
I think I prefer "CLI" tbh on the landing page—for reference, Stripe, Redis, Mongodb, Planetscale, Filebase, etc. all refer to their CLI as "CLI" (or preceded with the company name) within their docs page.

Reply:
(But if you're v set on that, I've opted for sentence case titles throughout docs!)

Reply:
Ah, makes sense. I wasn't sure how readable it was, but I'll defer to other docs pages.